Description: Link against libatomic for TBB check to prevent FTBFS on armel.
Author: Doug Torrance <dtorrance@debian.org>
Forwarded: https://github.com/Macaulay2/M2/pull/2528
Last-Update: 2022-06-14

--- a/M2/configure.ac
+++ b/M2/configure.ac
@@ -404,7 +404,8 @@
      AC_SUBST(LIBTBB)		dnl LIBTBB is usually set to -ltbb
      AC_LANG(C++)
      AC_CHECK_HEADER(tbb/tbb.h,
-	[AC_SEARCH_LIBS(TBB_runtime_interface_version,"tbb",
+	[LIBS="$LIBS -latomic"
+	 AC_SEARCH_LIBS(TBB_runtime_interface_version,"tbb",
 	   LIBTBB=$ac_cv_search_TBB_runtime_interface_version,
 	   [AC_MSG_ERROR([tbb library not found])])],
 	 [AC_MSG_ERROR([tbb include file (tbb.h) not found])])
